app/views/users/index.html.erb

```diff
@@ -1,2 +1,44 @@
-<h1>Users#index</h1>
-<p>Find me in app/views/users/index.html.erb</p>
+
+<section>
+  <% @users.each do |user| %>
```

Comment:
[OPTIONAL ] 
- Since you are already using patrials at _user.html.erb, using @users.each is not the best practice . The code snippet below should give you a clearer idea of what I am talking about.

   **app/views/users/_user.html.erb**

     ```
   HTML Code Here displaying
        user.photo, user.name and user.posts_counter
```

    **app/views/users/index.html.erb**

    `<%= render @users %>`